sdk/eventhub/event-processor-host/src/hostContext.ts

```diff
@@ -187,9 +187,6 @@ export namespace HostContext {
     // set defaults
     if (!options.consumerGroup) options.consumerGroup = defaultConsumerGroup;
     if (!options.eventHubPath) options.eventHubPath = config.entityPath;
-    if (!options.leaseRenewInterval)
```

Comment:
It looks like the `validateType` checks on lines 215/216 that check `options.leaseRenewInterval` and `options.leaseDuration` should be moved below where you set these options. This will then also validate the case where these values are set to the wrong type on `options.leaseManager`.